docker/image_types.yaml

```diff
@@ -3,17 +3,19 @@
 
 'base-image':
   tag: 'base-image'
-  context: 'docker/base-image'
+  dockerfile: 'docker/base-image/Dockerfile'
```

Comment:
Try to use a consistent order please. Alphabetical would be ideal.

Reply:
Done